controller/space.go

```diff
@@ -382,6 +387,16 @@ func ConvertSpaceFromModel(request *goa.RequestData, p space.Space, additional .
 					Related: &relatedCodebasesList,
 				},
 			},
+			Workitems: &app.RelationGeneric{
+				Links: &app.GenericLinks{
+					Related: &relatedWorkItemList,
+				},
+			},
+			Workitemlinktypes: &app.RelationGeneric{
```

Comment:
done in https://github.com/almighty/almighty-core/pull/1033/commits/a10c79a1a04b8618f4cde4fa216b44f6b663f225